pandas/tests/scalar/period/test_asfreq.py

```diff
@@ -656,6 +656,23 @@ def test_conv_secondly(self):
 
         assert ival_S.asfreq("S") == ival_S
 
+    def test_conv_microsecond(self):
+        # Avoid floating point errors dropping the start_time to before
+        #  the beginning of the Period
```

Comment:
Changes look good, `whatnew` entry notwithstanding.

Reference the PR number (since there is no issue number) here.